xcode/Subconscious/Shared/Services/DatabaseService.swift

```diff
@@ -829,6 +1069,7 @@ final class DatabaseService {
             sql: """
             SELECT id, title
             FROM memo
+            WHERE substr(memo.slug, 1, 1) != '_'
             ORDER BY RANDOM()
             LIMIT 1
             """
```

Comment:
`readRandomEntryLink` needs the same treatment as `readEntryBacklinks` etc. in the `compactMap` below. We're not rewriting the DIDs to Petnames for random suggestions at the moment.

Reply:
Oh, also, should random entries be scoped to the `owner`s sphere? I can see an argument to go either way.

If we _do_ allow random entries from peers then we need to update `pushRandomDetail` to use `MemoDetailDescription.from` so it uses the correct view.

Reply:
I think it may be best to scope to owner sphere.

Note that "random note" was a bit of a throw-away feature intended to generate some inspiration while we worked toward Geists. I would also be ok with removing it eventually.